source/cloud/azure/azure-mnmg.md

```diff
@@ -0,0 +1,70 @@
+# Multi-node, multi-GPU Cluster on Azure
+
+## Create a Cluster using Dask Cloud Provider
+
+The easiest way to setup a multi-node, multi-GPU cluster on Azure is to use [Dask Cloud Provider](https://cloudprovider.dask.org/en/latest/index.html).
+
+### 1. Install Dask Cloud Provider
+
+Dask Cloud Provider can be installed via `conda` or `pip`. The Azure-specific capabilities will need to be installed via the `[azure]` pip extra.
+
+```shell
+$ pip install dask-cloudprovider[azure]
+```
+
+### 2. Configure your Azure Resources
+
+Set up your Azure Resouce Group, Virtual Network, and Security Group according to [Dask Cloud Provider instructions](https://cloudprovider.dask.org/en/latest/azure.html#authentication).
+
+### 3. Create a Cluster
+
+From a Python terminal, a cluster can be created using the `dask_cloudprovider` package. The below example creates a cluster with 2 workers in `westus2` with `Standard_NC12s_v3` VMs. Ensure that the VMs have at least 100GB of disk.
```

Comment:
```suggestion
In Python, a cluster can be created using the `dask_cloudprovider` package. The below example creates a cluster with 2 workers in `westus2` with `Standard_NC12s_v3` VMs. Ensure that the VMs have at least 100GB of disk.
```

Reply:
Also might be useful to qualify why VMs need 100GB given we are directly asking users to set this. Generally this is to ensure there is space for the RAPIDS container image and associated dependencies.